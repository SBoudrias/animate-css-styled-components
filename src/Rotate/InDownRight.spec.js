import React from 'react';
import renderer from 'react-test-renderer';
import RotateInDownRight from './InDownRight';

it('renders with default props', () => {
	const tree = renderer.create(
		<RotateInDownRight />
	).toJSON();

	expect(tree).toMatchSnapshot();
});
