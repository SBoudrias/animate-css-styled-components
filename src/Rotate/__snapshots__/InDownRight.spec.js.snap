// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with default props 1`] = `
<div
  className="fDxOax"
  direction="normal"
  display="block"
/>
`;
